### ISSUE TITLE
AnalyticalTable: Table column headers not accessible via keyboard controls

### ISSUE THREAD

**Issue by angelzbg** [SAP/ui5-webcomponents-react#1889]
### Description
Table columns headers are not accessible via keyboard controls (ex. 'TAB' key)
https://sap.github.io/ui5-webcomponents-react/?path=/docs/data-display-analyticaltable--default-story

### Observed behavior: 
The Column headers of the table are not reachable via keyboard.
### Expected Behavior: 
The Column headers of the table should be reachable via keyboard.

### Steps to reproduce
1. Go to https://sap.github.io/ui5-webcomponents-react/?path=/docs/data-display-analyticaltable--default-story
2. Tab until focus on table headers
3. Table headers are not focused

### Context
Browser (if relevant): {...} Chrome

### Additional Context
Table headers can contain filters (ex: ascending, descending etc.) and they are not reachable.

[ACC-270.1_Central Business Configuration 2108_Table_column headers not reachable via keyboard](https://support.wdf.sap.corp/sap/bc/contentserver/001/ACC-270?get&pVersion=0046&contRep=CRMORDER&docId=0090FAE68E561EDBBBC727B00FF0810F&compId=ACC-270.1_Central%20Business%20Configuration%202108_Table_column%20headers%20not%20reachable%20via%20keyboard.docx&accessMode=r&authId=CN%3DBCS&expiration=20210728145235&secKey=MIHxBgkqhkiG9w0BBwKggeMwgeACAQExCzAJBgUrDgMCGgUAMAsGCSqGSIb3DQEHATGBwDCBvQIBATATMA4xDDAKBgNVBAMTA0JDUwIBADAJBgUrDgMCGgUAoF0wGAYJKoZIhvcNAQkDMQsGCSqGSIb3DQEHATAcBgkqhkiG9w0BCQUxDxcNMjEwNzI4MTI1MjM1WjAjBgkqhkiG9w0BCQQxFgQUAm7Pw6qoSZdDcMBdtOEuYAbenVswCQYHKoZIzjgEAwQuMCwCFBxELI5zxvrPnFJccT3cLyqksKQzAhQdEa85YVtFN4wYvuoUp%2BWFSUNw4A%3D%3D)

**Comment by MarcusNotheis** [comment]
Hey @angelzbg,

thanks for reporting! If I understood correctly, the bug is already reported in #1625, so I'm going to close this issue here.
In case I missed something please let me know and I'm happy to reopen the issue.
